src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -93,7 +93,9 @@ public ReportParser(Configuration configuration) {
                 LOG.log(Level.INFO, "File '{0}' does not contain features", jsonFile);
             }
             String jsonFileName = extractQualifier(jsonFile);
-            Arrays.stream(features).forEach(feature -> feature.setQualifier(jsonFileName));
+            Arrays.stream(features).forEach(feature ->
```

Comment:
did you test this code as well?

Reply:
I tested manually - but now added an integration test that covers this.